style.css

```diff
@@ -81,3 +108,185 @@
 .social-pic li {
   list-style-type: none;
 }
+
+.card {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  height: 552px;
+  width: 327px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 10px;
+  margin: auto;
+  margin-top: 100px;
+}
```

Comment:
- [ ] kindly consider sticking to the provided figma design to make a good implementation of it
 - consider decreasing the height you gave the technologies `li` 
 - also the margin your are giving this card make sure that both sides are equal check the images below for more clarification 

your design 👇  
<img width="458" alt="Screen Shot 2023-03-24 at 18 12 47" src="https://user-images.githubusercontent.com/60146030/227582586-d4fc2b12-00dd-4de5-81fe-c49f3b656109.png">

provided design 👇 
<img width="443" alt="Screen Shot 2023-03-24 at 18 19 26" src="https://user-images.githubusercontent.com/60146030/227582841-61e440d5-fbec-4a78-ac4d-735fc89f017c.png">